saplings/oauth-login/src/index.js

```diff
@@ -34,12 +34,12 @@ registerConfigSapling('login', () => {
     const accessTokenRegex = /\?access_token=([^&#]*)|&|$/;
     const token = window.$CANOPY.redirectedFrom.match(accessTokenRegex)[1];
     // Extract user information
-    const userIdRegex = /\?user_id=([^&#]*)|&|$/;
+    const userIdRegex = /&user_id=([^&#]*)|&|$/;
```

Comment:
These should not be expected in a particular order. You could match on the class `[\?|&]`. This would be true for all three Paramus.